kuka_kr10_support/package.xml

```diff
@@ -22,13 +24,22 @@
       unless noted otherwise (ie: no support for high speed joints,
       extended / limited motion ranges or other options).
     </p>
+    <p>
+      Joint limits and maximum joint velocities for the Agilus-2 variants
+      are based on the information in the <em>Kuka Deutschland GmbH - Kuka Robots
+      KR AGILUS-2 Spezifikation</em> version <em>Spez KR AGILUS-2 V4, 29.01.2019</em>.
+      All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise (ie: no support for high speed joints,
+      extended / limited motion ranges or other options).
+    </p>
     <p>
       Before using any of the configuration files and / or meshes included
       in this package, be sure to check they are correct for the particular
       robot model and configuration you intend to use them with.
     </p>
   </description>
   <author email="schindlbeck@imr.uni-hannover.de">Christopher Schindlbeck (University of Hanover)</author>
+  <author email="simon.schmeisser@isys-vision.de">Simon Schmeisser (isys vision GmbH)</author>
```

Comment:
Would this need to be updated?

Reply:
You're well informed! Optonic is the legal successor (and Isys Vision still a brand name) so nothing bad about the old name but you're right, yes